### PR TITLE
Re adds the recipes for Bicard, Kelotane, AntiTox, and Tricord

### PR DESCRIPTION
**Pull request by Sarcalogo**
Ported from

https://github.com/tgstation/-tg-station/pull/16515

### CODE REVIEW

**Comment by Crystalwarrior** [comment]
...why, though?
There are Goonchem equivalents to all the same chems save for Tricord, if anything we just need to buff goofchem variants.

I'm also against re-adding ez-regen Tricord chem.

@Kierany9 work on your GODDAMN CHEM UPDATE AAAAAAAAAA

**Comment by Sarcalogo** [comment]
The same justification in the /tg/ PR can be used here.

**Comment by Sarcalogo** [comment]
And no, those are all touch based.

**Comment by Sarcalogo** [comment]
Plus trek chem is cool.

**Comment by Kierany9** [comment]
>@Kierany9 work on your GODDAMN CHEM UPDATE AAAAAAAAAA

FUCK I'LL GET TO WORK ON IT SOON I PROMISE DONT BEAT ME PLEASE

File: code/modules/reagents/Chemistry-Recipes/Medicine.dm [hunk 1]
```diff
@@ -213,5 +213,31 @@
...
+	id = "antitoxin"
+	result = "antitoxin"
+	required_reagents = list("nitrogen" = 1, "silicon" = 1, "potassium" = 1)
+	result_amount = 3
+
+/datum/chemical_reaction/tricordrazine
+	name = "Tricordrazine"
+	id = "tricordrazine"
+	result = "tricordrazine"
+	required_reagents = list("bicardine" = 1, "kelotane" = 1, "antitoxin" = 1)
```
Comment (JohnGinnane) [inline code/modules/reagents/Chemistry-Recipes/Medicine.dm]: > bicardine

Should this be `bicaridine`?

**Comment by trifornt-tinfornt** [comment]
Honestly I'm not that large of a fan of this. I do want TrekChem back but honestly TrekChem needs some changes as any Chemist willing to put in the time could literally become a supersonic godmode chemist that would annoy everyone around them.

**Comment by McBawbaggings** [comment]
I like trekchem, but we need overdose things for them

**Comment by Sarcalogo** [comment]
There already exists overdose thresholds for all of these drugs that are still in the game, just not constructable via chem dispenser.

**Comment by calibraptor** [comment]
+1

**Comment by Nebyab** [comment]
yeah sure, waiting on a codecheck

**Comment by dreamcrab** [comment]
i don't really care either way, i think this could get messy with people mixing healing chems if they don't count as their counterparts for overdoses.

**Comment by Kokojos** [comment]
Headmins said yes

**Pull request merged**